common/src/main/java/com/viaversion/viarewind/protocol/v1_8to1_7_6_10/data/VirtualHologramEntity.java

```diff
@@ -111,6 +112,7 @@ public void syncState(final EntityPacketRewriter1_8 entityRewriter, final List<E
         final boolean invisible = (flags & 0x20) != 0;
         small = (armorStandFlags & 0x01) != 0;
         marker = (armorStandFlags & 0x10) != 0;
+        sneaking = (flags & 0x02) != 0;
```

Comment:
Is this tested? Sneaking is flag 1 and not 2 afaik.


I also wonder what this does. Can you maybe provide a video showing how the client reacts when this entity data is set? I can't find an implementation for it in the Minecraft sourcecode